### PR TITLE
Some Butter project conversion from old project

### PR DESCRIPTION
**Pull request by hadim**
- Butter app easly build without error
- Butter app start with `grunt start`
- Torrents from Vodo can load but can't be played since the format is not supported
- All others torrents or magnet links I have tried stay stucked on `[INFO] Loading torrent`
- Parameters button does not work : 

```
[ERROR] TypeError: Cannot read property 'authenticated' of null TypeError: Cannot read property 'authenticated' of null
    at eval (eval at <anonymous> (app://host/src/app/vendor/underscore/underscore.js:1454:20), <anonymous>:272:23)
    at template (app://host/src/app/vendor/underscore/underscore.js:1461:21)
    at Object.Marionette.Renderer.render (app://host/src/app/vendor/marionette/lib/backbone.marionette.js:1655:14)
    at Marionette.ItemView.Marionette.View.extend._renderTemplate (app://host/src/app/vendor/marionette/lib/backbone.marionette.js:2117:38)
    at Marionette.ItemView.Marionette.View.extend.render (app://host/src/app/vendor/marionette/lib/backbone.marionette.js:2085:12)
    at Marionette.Region.Marionette.Object.extend._renderView (app://host/src/app/vendor/marionette/lib/backbone.marionette.js:1210:12)
    at Marionette.Region.Marionette.Object.extend.show (app://host/src/app/vendor/marionette/lib/backbone.marionette.js:1138:14)
    at Backbone.Marionette.LayoutView.extend.showSettings (app://host/src/app/lib/views/main_window.js:419:27)
    at triggerEvents (app://host/src/app/vendor/backbone/backbone.js:368:57)
    at triggerApi (app://host/src/app/vendor/backbone/backbone.js:356:19)
undefined:272 Uncaught TypeError: Cannot read property 'authenticated' of null
```

### CODE REVIEW

**Comment by hadim** [comment]
The most important feature that does not work is the fact custom torrents never start downloading and so can't be played... I don't know why.

File: Gruntfile.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ module.exports = function (grunt) {
                 cmd: '"build/cache/linux32/<%= nwjs.options.version %>/nw" .'
             },
             linux64: {
-                cmd: '"build/cache/linux64/<%= nwjs.options.version %>/nw" .'
+                cmd: '"build/' + projectName + '/linux64/' + projectName + '" .'
             },
```
Comment (xaiki) [inline Gruntfile.js]: why ?

File: Gruntfile.js [hunk 2]
```diff
@@ -464,7 +464,7 @@ module.exports = function (grunt) {
         },
 
         clean: {
-            releases: ['build/releases/' + projectName + '/**'],
+            releases: ['build/' + projectName + '/**'],
```
Comment (xaiki) [inline Gruntfile.js]: why ?

File: Gruntfile.js (hunk 1 above)
Comment (hadim) [inline Gruntfile.js]: i don't why its the path generated on my system. can you check with a fresh npm install ??

**Comment by hadim** [comment]
```
 $ tree build/
build/
└── Butter
    └── linux64
        ├── Butter
        ├── icudtl.dat
        ├── libffmpegsumo.so
        ├── locales
        │   ├── am.pak
        │   ├── ar.pak
        │   ├── bg.pak
        │   ├── bn.pak
        │   ├── ca.pak
        │   ├── cs.pak
        │   ├── da.pak
        │   ├── de.pak
        │   ├── el.pak
        │   ├── en-GB.pak
        │   ├── en-US.pak
        │   ├── es-419.pak
        │   ├── es.pak
        │   ├── et.pak
        │   ├── fa.pak
        │   ├── fil.pak
        │   ├── fi.pak
        │   ├── fr.pak
        │   ├── gu.pak
        │   ├── hi.pak
        │   ├── hr.pak
        │   ├── hu.pak
        │   ├── id.pak
        │   ├── it.pak
        │   ├── iw.pak
        │   ├── ja.pak
        │   ├── kn.pak
        │   ├── ko.pak
        │   ├── lt.pak
        │   ├── lv.pak
        │   ├── ml.pak
        │   ├── mr.pak
        │   ├── ms.pak
        │   ├── nl.pak
        │   ├── no.pak
        │   ├── pl.pak
        │   ├── pt-BR.pak
        │   ├── pt-PT.pak
        │   ├── ro.pak
        │   ├── ru.pak
        │   ├── sk.pak
        │   ├── sl.pak
        │   ├── sr.pak
        │   ├── sv.pak
        │   ├── sw.pak
        │   ├── ta.pak
        │   ├── te.pak
        │   ├── th.pak
        │   ├── tr.pak
        │   ├── uk.pak
        │   ├── vi.pak
        │   ├── zh-CN.pak
        │   └── zh-TW.pak
        └── nw.pak

3 directories, 57 files
```

**Pull request merged**